aeron-tools/src/main/java/uk/co/real_logic/aeron/tools/PublisherTool.java

```diff
@@ -0,0 +1,495 @@
+/*
+ * Copyright 2015 Kaazing Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.co.real_logic.aeron.tools;
+
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+
+import org.apache.commons.cli.ParseException;
+import org.slf4j.Logger;
```

Comment:
Will be moving the subproject aeron-tools out of top-level build.gradle and moving to aeron-tools/build.gradle and moving third-party dependencies into the aeron-tools/build.gradle and out of the top-level build.gradle. We will later probably go back and swap out the SLF with java.util.logging.